modules/phase_field/test/tests/phase_field_kernels/SplitCahnHilliard.i

```diff
@@ -64,16 +61,13 @@
   [./pfmobility]
     type = GenericConstantMaterial
     prop_names  = 'M kappa_c'
-    prop_values = '1e-3 0.1'
+    prop_values = '100 40'
   [../]
 
   [./free_energy]
-    type = DerivativeParsedMaterial
+    type = MathFreeEnergy
```

Comment:
I am not crazy about replacing the DerivativeParsedMaterial here with MathFreeEnergy- the reason is because this test is probably the best place for a new MOOSE user to look and see how to put together a split Cahn-Hilliard model with arbitrary free energy, which is where a lot of people are going to want to start when beginning to use the phase-field model. Can you use the DerivativeParsedMaterial to put in the MathFreeEnergy analytical energy expression rather than using the MathFreeEnergy material?

Reply:
Yeah, I can do that. The motivation was to have it similar to the AD version of that test. But I can just put a comment in.

Reply:
Done